integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -32,22 +32,23 @@ This enhanced support gives you greater flexibility and choice in managing your
 
 You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners:
 
-* {product-title-short} Scanner (recommended)
+* {product-title-short} Scanner V4: Beginning with release 4.4, a new scanner is introduced that is built on the {product-title-short} StackRox and Clair scanners. The new scanner, Scanner V4, supports scanning of language and OS-specific image components. You do not have to create an integration to use this scanner, but you must enable it during or after installation. You must use this scanner in conjunction with the StackRox Scanner. See the installation documentation for more information.
```

Comment:
I don't believe we are prefixing this with RHACS. Instead, just `Scanner V4`. Let's confirm with Shubha offline

Reply:
> built on the {product-title-short} StackRox and Clair scanners

That's not quite right. It is built on top of [ClairCore](https://github.com/quay/claircore) ([alternate link](https://quay.github.io/claircore/)) which also powers [Clair](https://github.com/quay/clair) ([alternate link](https://quay.github.io/clair/))

Reply:
> You must use this scanner in conjunction with the StackRox Scanner

Maybe make it clear this is only required at this time, and this requirement is because StackRox Scanner has the node and platform scanning capabilities which Scanner V4 currently lacks. For image scanning, only, StackRox Scanner is not required